lib/filter-pack.js

```diff
@@ -3,7 +3,7 @@
 var paethPredictor = require('./paeth-predictor');
 
 function filterNone(pxData, pxPos, byteWidth, rawData, rawPos) {
-  pxData.copy(rawData, rawPos, pxPos, pxPos + byteWidth);
```

Comment:
If you look at the other filter methods (not sum methods), it copies into rawData and returns nothing.
So this doesn't look right.